scipy/ndimage/tests/test_filters.py

```diff
@@ -3,6 +3,7 @@
 import itertools
 import math
 import numpy
+import numpy as np
```

Comment:
Should we either just stick with numpy or change numpy->np everywhere in this file?

Reply:
To avoid a larger diff.
```suggestion
import numpy
```